### DISCUSSION TITLE
My test ran a few steps and quit.

### DISCUSSION THREAD

**Discussion by KangChenRui** [idaholab/moose#20983] [Q&A General] [answered]
Hello,everyone!

My test ran a few steps and quit.

The following is the report information：

```
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   PID 33867 RUNNING AT cu03
=   EXIT CODE: 9
=   CLEANING UP REMAINING PROCESSES
=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES
===================================================================================
YOUR APPLICATION TERMINATED WITH THE EXIT STRING: Killed (signal 9)
This typically refers to a problem with your application.
Please see the FAQ page for debugging suggestions
```
Can you help me?

Chen Rui

**Comment by GiudGiud** [comment, marked as answer]
Hello

this isn’t enough information to see what went wrong. 
could you please use a debugger to get us a backtrace? See this page for instructions

https://mooseframework.inl.gov/application_development/debugging.html

guillaume

**Reply by GiudGiud** [reply]
Also could you please describe the test a little bit. What does it? Any parallelism? Is it all moose objects or is there custom ones from your app?

**Reply by GiudGiud** [reply]
@KangChenRui any update on this?

**Reply by KangChenRui** [reply]
Hi, GiudGiud
I am trying to debug with debugger.
Thanks for your reply!

**Reply by KangChenRui** [reply]
Hi, GiudGiud
The following is the background output information corresponding to the inputfile.
Can you help me?
Thanks for your reply!
[nohup.out.txt](https://github.com/idaholab/moose/files/8752056/nohup.out.txt)

**Reply by GiudGiud** [reply]
signal 9 (at the end of the log) is sigkill.
 Did you kill the process?
Or are you running out of time on a cluster session?
Another possiblity is that you are running out of memory. Can you try to reduce the problem and see if it runs?
Also there is a postprocessor to keep track of memory usage:
https://mooseframework.inl.gov/source/postprocessors/MemoryUsage.html

The log seems fine except for this sigkill

**Reply by Adairle** [reply]
EnvironmentNameNotFound: Could not find conda environment: moose
You can list all discoverable environments with `conda info --envs`.
That's it
Look forward to your reply, thank you！

**Reply by GiudGiud** [reply]
You should re-install the environment. 
https://mooseframework.inl.gov/getting_started/installation/conda.html
